packages/eas-cli/src/build/types.ts

```diff
@@ -12,3 +12,11 @@ export enum BuildDistributionType {
   INTERNAL = 'internal',
   SIMULATOR = 'simulator',
 }
+
+export enum BuildResourceClass {
```

Comment:
you are using this type in some places and in others, you are importing a copy of it from eas-build-job, it should be more consistent in either direction